### PR TITLE
i18n(zh-cn): Add 5 error translations

### PR DESCRIPTION
**Pull request by XindiShang**
#### What kind of changes does this PR include?

- Translated content

#### Description

Added 5 Astro error translations:
- `unknown-compiler-error.mdx`
- `client-address-not-available.mdx`
- `missing-media-query-directive.mdx`
- `no-matching-renderer.mdx`
- `no-client-entrypoint.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/unknown-compiler-error.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<DontEditWarning />
+
+
+> 未知的编译器错误。 (E01000)
+
+## 哪里出了问题？
+Astro 在编译文件时遇到了未知错误。在大多数情况下，这不是你的错，而是编译器的问题。
+
+如果现有的 issue 中没有提到这个问题，请[创建一个 issue](https://astro.build/issues/compiler)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/unknown-compiler-error.mdx:16]: The compiler issues url returned 404 page.

I create a pr in English page.
 https://github.com/withastro/docs/pull/2838

File: src/content/docs/zh-cn/reference/errors/no-client-entrypoint.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **NoClientEntrypoint**: `COMPONENT_NAME` 组件包含了 `client:CLIENT_DIRECTIVE` 客户端指令，但是 `RENDERER_NAME` 渲染器没有提供客户端入口点。 (E03008)
+
+
+## 哪里出了问题？
+Astro 尝试在客户端对组件进行激活，但是所使用的渲染器没有提供用于激活的客户端入口点。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/no-client-entrypoint.mdx:15]:
```suggestion
Astro 尝试在客户端对组件进行激活，但是所使用的渲染器没有提供用于激活的客户端入口文件。
```

File: src/content/docs/zh-cn/reference/errors/unknown-compiler-error.mdx (hunk 1 above)
Comment (liruifengv) [reply src/content/docs/zh-cn/reference/errors/unknown-compiler-error.mdx:16]: It fixed.

**Review by liruifengv — APPROVED** [review]
Chinese LGTM.

**Review by Yan-Thomas — APPROVED** [review]
LGTM, thanks @XindiShang & @liruifengv! 🙌

**Pull request merged**